NYTPhotoViewer/NYTPhotoCaptionView.m

```diff
@@ -72,7 +72,8 @@ - (void)layoutSubviews {
 - (CGSize)intrinsicContentSize {
     CGSize contentSize = [self.textView sizeThatFits:CGSizeMake(self.preferredMaxLayoutWidth, CGFLOAT_MAX)];
     CGFloat width = (CGFloat)self.preferredMaxLayoutWidth;
-    CGFloat height = (CGFloat)ceil(contentSize.height);
+    CGFloat verticalPadding = 40.0;
```

Comment:
What does this vertical padding reflect? How does it affect other, non X iPhones?

Reply:
Without padding, the caption text gets cut off on non X phones.
<img width="559" alt="screen shot 2017-11-06 at 2 13 46 pm" src="https://user-images.githubusercontent.com/8617759/32459253-00060014-c2fd-11e7-81fe-257dde563764.png">